packages/hooks/README.md

```diff
@@ -41,4 +41,11 @@ Hooks can be added to an object via composition:
 
 `myObject.hooks = createHooks();`
 
-API functions are then be called: `myObject.hooks.addAction()`...
+API functions are then be called: `myObject.hooks.addAction()`.
+
+### Events on action/filter add or remove.
+
+Whenever an action or filter is added or removed, a matching `hookAdded` or `hookRemoved` action is triggered.
+
+* `hookAdded` is triggered when `hooks.addFilter` or `hooks.addAction` is called, passing values for `hookName`, `functionName`, `callback` and `priority`.
```

Comment:
What about rephrasing to:

> `hookAdded` action is triggered when `addFilter()` or `addAction()` method is called, passing values for `hookName`, `functionName`, `callback` and `priority`.

A similar change could be applied to the 2nd line.